apps/design-system/registry/default/example/drawer-dialog.tsx

```diff
@@ -1,3 +1,5 @@
+'use client'
```

Comment:
I don’t fully get why we need to use the `client` directive now (and not before) but it seems necessary for the build to work.

Reply:
Don't think it necessarily needs it, I put it there while debugging, can remove it.